### DISCUSSION TITLE
Please activate my API key

### DISCUSSION THREAD

**Discussion by aadilmallick** [etsy/open-api#1250] [Q&A]
I requested a personal access API key and it's been nearly two weeks and I still haven't had any response from etsy. You'd think things like this would be automated, but ig not.

**Comment by constantin-bogdan** [comment]
I have the same problem, they need like 20+ days to approve your app request. Etsy doenst make it easy for developers to use their API. So disappointing.

**Comment by aadilmallick** [comment]
If they don't give it to me by the end of the month I am going to scrape the hell out of their website with Selenium

**Reply by madalinignisca** [reply]
I hope u can afford the amount of servers to be able to even double their response time per requests...

**Reply by aadilmallick** [reply]
nvm they banned my account lol

**Comment by Wilderbill** [comment]
Waiting for approval (3 weeks) is the easy part.
Being denied because you made an application mistake and having to wait another three weeks,
that is when it really gets frustrating.
